lighthouse-cli/cli-flags.js

```diff
@@ -64,6 +64,7 @@ function getFlags(manualArgv) {
         ],
         'Configuration:')
       .describe({
+        // We don't allowlist specific locales. Why? So we can support the user who requests 'es-MX' (unsupported) and we'll fall back to 'es' (supported)
```

Comment:
we might want a command for listing available locales, but something for another day :)